BugFixes/README.md

```diff
@@ -15,16 +14,13 @@ exported variables may fail. Enforce this in the compiler.
 Fix 2: Initializing of `GraphicFrames.TBuf` variable
 ----------------------------------------------------
 
-
-When you delete text from a Drawing, the extra letters get moved into a `Texts.Buffer` called `Tbuf`.
+If you delete text from a Drawing, the extra letters get moved into a `Texts.Buffer` called `Tbuf`.
```

Comment:
When/If in this case (general condition, "whenever") is not a typo, but a matter of style. Compare https://www.bbc.co.uk/worldservice/learningenglish/grammar/learnit/learnitv261.shtml#:~:text=when%20and%20if%20are%20interchangeable,general%20conditions